tests/test_CtfEstimate.py

```diff
@@ -54,22 +51,31 @@ def testEstimateCTF(self):
                     g_max=5.0,
                     output_dir=tmp_output_dir,
                     dtype=np.float64,
+                    save_ctf_images=True,
+                    save_noise_images=True,
                 )
 
                 logger.debug(f"results: {results}")
 
-                for i, result in enumerate(results):
-                    diffs = np.subtract(result[:-1], self.test_output[i][:-1])
-                    logger.debug(f"diffs: {diffs}")
+                for result in results.values():
+                    # the following parameters have higher tolerances
 
                     # defocusU
-                    np.allclose(result[0], self.test_output[i][0], atol=5e-2)
+                    np.allclose(
+                        result["defocus_u"], self.test_output["defocus_u"], atol=5e-2
```

Comment:
These `atol`s due to the indeterminancy of the algorithm? Would be good to put a note.

Reply:
That I'm not sure about it. Every time I run the test and peek at the CTF estimated, the numbers are exactly the same down to like `1e-10`.